src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -205,6 +207,11 @@ struct Input
         "Number of contour repetitions to run for each MarchingCubes object")
       ->capture_default_str();
 
+    app.add_option("--maskCount", maskCount)
+      ->description(
+        "Split the zones up using this many masks (default to 1).")
```

Comment:
>"Split the zones up using this many masks (default to 1).")

I don't understand this -- what does it mean to split a zone using a mask? 
Please expand/enhance this description.

Reply:
Also, is there a range of validity? E.g. does this need to be non-zero? non-negative? is there an upper range? If so, please add appropriate checks/validators (https://github.com/CLIUtils/CLI11/blob/main/README.md#validators)

Reply:
Done.  Thank you.

Reply:
I rephrased the help message.